spec/person_spec.rb

```diff
@@ -0,0 +1,44 @@
+require_relative '../person'
+
+describe Person do
+  context 'When a student or teacher is created' do
+    it 'takes the age and name and returns an instance of the person' do
+      person = Person.new(28, 'Regina', 'True')
+      expect(person).to be_instance_of(Person)
+    end
+
+    it 'should have attributes' do
+      person = Person.new(28, 'Regina', 'True')
+      expect(person).to have_attributes(age: 28)
+      expect(person).to have_attributes(name: 'Regina')
+      expect(person).to have_attributes(parent_permission: 'True')
+    end
+
+    it 'the methods should work' do
+      person = Person.new(28, 'Regina', 'True')
+      expect(person.can_use_services?).to be true
+    end
+  end
+end
+
+RSpec.describe Person do
+  let(:person) { Person.new(28, 'Regina', 'True') }
+
+  describe '#can_use_services?' do
+    context 'when the person is of age' do
+      it 'returns true' do
+        person.age = 28
+        expect(person.can_use_services?).to eq(true)
+      end
+    end
+  end
+
+  describe '#of_age?' do
+    context 'when the person is of age' do
+      it 'returns true' do
+        person.age = 28
+        expect(person.send(:of_age?)).to eq(true)
+      end
+    end
```

Comment:
- [ ] Good job adding these tests as requested by the previous reviewer, however, you are still missing one test mentioned. Kindly add tests for the `add_rental` method in the Person class.